code/index.html

```diff
@@ -1,22 +1,79 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
-
-    <section class="">
-      <!-- The other news  -->
-    </section>
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Project 2: The News Site</title>
+  <link rel="stylesheet" href="./style.css" />
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans+Condensed:wght@300&display=swap" rel="stylesheet">
+  <!-- dont forget the meta tag for responsiveness -->
+</head>
+
+<body>
+  <header>
+    <!-- Some kind of logo and navbar -->
+    <nav>
+      <div class=smiley>
+        <img class="logo" src="Smiley.png" alt=smiley>
+        <h1>The upBeat</h1>
+      </div>
+      <div class="hamburger">
+        <a href="/html/">Start</a>
+        <a href="/css/">Today</a>
+        <a href="/js/">Random</a>
+        <a href="/python/">Contact</a>
+      </div>
+    </nav>
+  </header>
+
+  <section class="big-news">
+    <article class=header>
+      <h2>The world hasn't ended!</h2>
+      <p>At least we have that going for us. Here we offer a respite from the news of darkness and despair and give you
+        upBeat news!</p>
+    </article>
+  </section>
+
+  <hr>
+
+  <section class="small-news">
+    <article class=smallstory>
+      <img class=articleimage src="least tern.jpeg" alt=least tern>
+      <div class=articletext>
```

Comment:
class names

Reply:
Are they too long? Not descriptive enough?

Reply:
You need to put "" around, and go for kebab-case or camelCase. so instead of `smallstory` I would go for `"small-story"` or `"smallStory"` The importance is the " around